### DISCUSSION TITLE
Scrub not run but enabled in script.

### DISCUSSION THREAD

**Discussion by snakekick** [auanasgheps/snapraid-aio-script#68] [Q&A]
Hi there,
I think I have scrubbing enabled in the config script but no scrub job run.

SCRUB_PERCENT=5
SCRUB_AGE=10

Or does scrub age 10 days mean that the scrub will only run every 10 days?

**Comment by auanasgheps** [comment]
Hi, 

Age 10 does NOT mean it will run every 10 days.

SCRUB_PERCENT: amount of the array to be scrubbed
SCRUB_AGE: data not scrubbed in the previous (10) days 

To enable scrub every N days, use SCRUB_DELAYED_RUN. 

How do you tell it does not run scrub?

**Comment by snakekick** [comment]
Hello @auanasgheps 
i see this in the daily report:

SnapRAID SCRUB [Tue Aug 29 09:25:31 AM CEST 2023]
SCRUB Previous Blocks [Tue Aug 29 09:25:31 AM CEST 2023]
Self test...  
Loading state from /mnt/disk1/snapraid.content...  
WARNING! With 5 disks it's recommended to use two parity levels.  
Using 602 MiB of memory for the file-system.  
Initializing...  
Using 112 MiB of memory for 64 cached blocks.  
Selecting...  
Scrubbing...  
SCRUB - Nothing to do

SCRUB finished [Tue Aug 29 09:25:42 AM CEST 2023]

**Reply by auanasgheps** [reply]
What happens if you simply execute `snapraid scrub` in a shell on your server? I have the feeling has something to do with snapraid itself.
